### PR TITLE
Card refactor

### PR DESCRIPTION
**Pull request by Sharmaryan**
- refactored card

### CODE REVIEW

File: styles/card/card.css [hunk 1]
```diff
@@ -20,12 +21,13 @@
...
   font-weight: bold;
   padding: 0.4rem;
-  margin: 0.1rem;
+  margin: 0.1rem 0; 
   cursor: pointer;
 }
 .card-logo {
-  height: 150px;
-  width: 200px;
+  /* height: 150px; */
```
Comment (chanchal-panpaliya) [inline styles/card/card.css]: remove comment

File: styles/card/card.html [hunk 2]
```diff
@@ -180,7 +181,8 @@ <h3 class="inner-container-types">vertical card</h3>
         </p>
         <div class="card-container component-container">
           <div class="card card-vertical">
-            <img src="car.png" alt="" class="card-logo" />
+            <!-- <img src="car.png" alt="" class="card-logo" /> -->
```
Comment (chanchal-panpaliya) [inline styles/card/card.html]: remove comment

File: styles/card/card.html [hunk 3]
```diff
@@ -180,7 +181,8 @@ <h3 class="inner-container-types">vertical card</h3>
...
+            <img src="https://cdn.pixabay.com/photo/2015/04/23/22/00/tree-736885__480.jpg" alt="" class="card-logo" />
```
Comment (chanchal-panpaliya) [inline styles/card/card.html]: alt is empty

**Review by chanchal-panpaliya — APPROVED** [review]
code looks clean with proper naming with good UI

**Review by Kevin-Solomon — APPROVED** [review]

**Pull request merged**